### ISSUE TITLE
Update X3ML schema to support mappings from terminologies

### ISSUE THREAD

**Issue by ymark** [isl/x3ml#99]
Update the schema of X3ML to support the new syntax for terminologies. More specifically at least the following should be supported:

* broader (for broader concepts in a terminology)
* exactMatch (for exact match concepts in a terminology)

**Comment by ymark** [comment]
X3ML schema has been updated by adding 2 new elements described below under the **if** element.

## New elements: 

### Broader

Used for retrieving the broader terms from a SKOS taxonomy. It has the following syntax:

```
<broader value="VASE">type/text()</broader>
```

### ExactMatch

Used for retrieving the exact match terms from a SKOS taxonomy. It has the following syntax:

```
<exact_match value="VASE">type/text()</exact_match>
```

**Comment by ymark** [comment]
The new version of the X3ML schema is 1.3 and it will be made publicly available from X3ML Engine version 1.8.2 onwards
